### PR TITLE
fix(966): Remove auth for GET banners endpoints

### PR DESCRIPTION
**Pull request by tkyi**
## Context

Users shouldn't need to login to Screwdriver to see banner announcements.

## Objective

This PR removes auth for GETs to banners endpoints.

## References

Related links: https://github.com/screwdriver-cd/screwdriver/issues/966

### CODE REVIEW

File: plugins/banners/get.js [hunk 1]
```diff
@@ -13,10 +13,6 @@ module.exports = () => ({
         description: 'Get a single banner',
         notes: 'Return a banner record',
         tags: ['api', 'banners'],
-        auth: {
-            strategies: ['token'],
-            scope: ['user', '!guest']
-        },
         plugins: {
             'hapi-swagger': {
                 security: [{ token: [] }]
```
Comment (jithin1987) [inline plugins/banners/get.js]: should this be removed ?

File: plugins/banners/list.js [hunk 2]
```diff
@@ -10,10 +10,6 @@ module.exports = () => ({
         description: 'Get banners',
         notes: 'Returns all banner records',
         tags: ['api', 'banners'],
-        auth: {
-            strategies: ['token'],
-            scope: ['user']
-        },
         plugins: {
             'hapi-swagger': {
                 security: [{ token: [] }]
```
Comment (jithin1987) [inline plugins/banners/list.js]: should this be removed ?

File: plugins/banners/get.js (hunk 1 above)
Comment (tkyi) [reply plugins/banners/get.js]: removed 👍

File: plugins/banners/list.js (hunk 2 above)
Comment (tkyi) [reply plugins/banners/list.js]: removed

**Pull request merged**